elab_expr.cc

```diff
@@ -4030,7 +4030,7 @@ static verinum param_part_select_bits(const verinum&par_val, long wid,
 	// If the input is a string, and the part select is working on
 	// byte boundaries, then make the result into a string.
       if (par_val.is_string() && (labs(lsv)%8 == 0) && (wid%8 == 0))
-	    return result.as_string();
+	    return verinum(result.as_string());
```

Comment:
From a type perspective that would be correct, but from a functional standpoint it would not be. We created a numeric verinum at the beginning of the routine and that is being converted to a string verinum at this line. The string verinum constructor is the only one that can set the string type to true. Using just "return result" would return a numeric value which is incorrect in the context of a parameter wanting a string value.

Everything else looks good and takes something off my to do list.

Thanks